spice/deps/helper_cb.py

```diff
@@ -27,12 +27,12 @@
 import os
 import logging
 import argparse
+from PIL import Image, ImageDraw
```

Comment:
No Image and ImageDraw are modules (see http://effbot.org/imagingbook/image.htm, http://effbot.org/imagingbook/image.htm, http://effbot.org/imagingbook/imagedraw.htm). PIL is library. It works on VMs in the bkr02 machine without any additional pip installation